assests/CSS/mobileStyle.css

```diff
@@ -0,0 +1,558 @@
+  line-height: 23px;
+  padding: 0 1.5vw;
+}
+
+.headline .primary-text.about {
+  text-align: center;
+  padding: 0 4.5vw;
+}
+
+.headline .secondary-text {
+  font-weight: 300;
+  font-size: 3vw;
+  line-height: 17px;
+  padding: 10px 1vw;
+  background-color: aliceblue;
+}
+
+.headline .secondary-text.about {
+  padding: 0 0.5vw;
+}
+
+.headline .date {
+  font-weight: 200;
+  font-size: 7vw;
+  color: #272a31;
+}
+
+.headline .venue {
+  padding: 10px 5px;
+  font-weight: 400;
+  font-size: 5vw;
+  color: #272a31;
+}
+
+.web-address {
+  display: block;
+  padding: 10px 15px;
+  font-weight: 400;
+  font-size: 5vw;
+}
+
+.headline .contact-us {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 1.5vw;
+  line-height: 2.5vw;
+  text-align: center;
+  margin: 0 auto;
+  padding: 0 7.5vw;
+}
+
+.headline .email {
+  text-decoration: underline;
+  font-family: 'lato', sans-serif;
+  font-weight: 700;
+  color: #272a31;
+}
+
+/* main program card section */
+.main-program {
+  display: flex;
+  flex-direction: column;
+  background-image: url(../Images/program_bkgr_mob.png);
+  color: #fff;
+  padding: 7vw;
+}
+
+.main-heading {
+  position: relative;
+  margin-bottom: 10%;
+  font-family: 'lato', sans-serif;
+  font-weight: 600;
+  font-size: 5vw;
+  text-align: center;
+}
+
+.main-heading::after {
+  background: #ec5242;
+  content: '';
+  width: 10%;
+  height: 2px;
+  left: 43%;
+  bottom: -10px;
+  position: absolute;
+}
+
+.program-articles {
+  display: flex;
+  flex-direction: column;
+  gap: 3vw;
+}
+
+.main-program article {
+  display: flex;
+  padding: 3%;
+  gap: 5px;
+  align-items: center;
+  border: none;
+  background-image: url(../Images/article-program-mob.png);
+}
+
+.main-program .fa {
+  font-size: 7vw;
+}
+
+.fa img {
+  width: 25px;
+  height: 30px;
+}
+
+.program-title {
+  font-family: 'lato', sans-serif;
+  font-weight: 400;
+  font-size: 4vw;
+  color: #ec5242;
+  padding-left: 5px;
+}
+
+.program-description {
+  font-family: 'lato', sans-serif;
+  font-weight: 400;
+  font-size: 2.8vw;
+  color: #fff;
+  width: 60%;
+}
+
+.see-more-mobile {
+  font-family: 'lato', sans-serif;
+  font-weight: 400;
+  font-size: 4vw;
+  color: #fff;
+  display: block;
+  text-align: center;
+  text-decoration: none;
+  background: #ec5242;
+  padding: 10% 3%;
+  width: 80%;
+  margin: 12% auto;
+}
+
+.see-more-desktop {
+  display: none;
+}
+
+/* Green legacy featured events section */
+.gl-events {
+  background: #fff;
+}
+
+.gl-title {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 6vw;
+  text-align: center;
+  padding: 5% 0;
+  margin-bottom: 2%;
+  position: relative;
+}
+
+.gl-title::after {
+  content: '';
+  width: 10%;
+  height: 3px;
+  background: #ec5242;
+  left: 41%;
+  bottom: 5px;
+  position: absolute;
+}
+
+.events {
+  display: grid;
+  grid-template-columns: 80%;
+  grid-auto-rows: auto;
+  row-gap: 2rem;
+  justify-content: center;
+  margin-bottom: 2rem;
+}
```

Comment:
- [ ] **[Optional]** **Congratulations on implementing a good project :clap::clap:** however, It would be better for the best user experience to increase the ```font-size``` of your text on a mobile view it's, not clearly readable. :pray: :pray: 
- [ ] **[Optional]** and Kindly try to make sure that the pictures on the mobile are not compressed to make your design more professional :+1: :+1: 

**Your design** 
![image](https://user-images.githubusercontent.com/102757126/205444499-b8a818d6-2d7d-44ce-9fae-0fd465b4c422.png)